### PR TITLE
Test observability

### PR DESCRIPTION
**Pull request by Yatin2108**
Test observability added

### CODE REVIEW

**Review by princebaretto99 — CHANGES_REQUESTED** [review]
Do we need the Design.MD file?

**Review by princebaretto99 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -18,12 +18,12 @@
 		"bstack-local-parallel-multiple": "nightwatch -c resources/conf/runners/browserstack_local.js --env  browserstack.firefox,browserstack.chrome,browserstack.edge --workers=5"
 	},
 	"devDependencies": {
-		"@nightwatch/browserstack": "^0.1.3",
+		"@nightwatch/browserstack": "^0.2.1",
```
Comment (samirans89) [inline package.json:21]: why woudn't we use the latest version? https://www.npmjs.com/package/@nightwatch/browserstack/v/3.1.1

File: package.json [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
 		"allure-commandline": "2.13.8",
 		"browserstack-local": "1.4.8",
```
Comment (samirans89) [inline package.json:23]: do we need this explicitly?

`@nightwatch/browserstack` already includes browserstack-local

File: package.json [hunk 3]
```diff
@@ -18,12 +18,12 @@
...
 		"chromedriver": "^90.0.0",
```
Comment (samirans89) [inline package.json:24]: can we change this to latest chrome, if being used?

File: package.json [hunk 4]
```diff
@@ -18,12 +18,12 @@
...
 		"eslint": "7.21.0",
-		"nightwatch": "^2.6.15",
+		"nightwatch": "^2.6.21",
```
Comment (samirans89) [inline package.json:26]: latest version: 3.1.3

**Review by samirans89 — APPROVED** [review]

**Pull request merged**